README.md

```diff
@@ -1,153 +1,169 @@
+* **Cluster Type**: Docker [Unified Control Plane](https://www.docker.com/products/docker-universal-control-plane) | Docker [Swarm](https://docs.docker.com/swarm/)
+* **Cluster OS**: [CentOS 7.2](https://www.centos.org/)
+* **Cluster Target**: Bare Metal | [Vagrant](https://www.vagrantup.com/)
 
-##Definitions and Terms:
-###Server/Node/Host/Asset
-A standalone physical/virtual machine with predefined/discovered compute, memory, storage
+##Definitions
+To better understand Cluster Manager, you should know a few system definitions.
+
+###Node
+A physical or virtual server with predefined/discovered compute, memory, storage
 and networking capabilities.
 
 ###Cluster
-A collection of one or more nodes. Nodes may or may not be homogeneous in their capabilities.
+A collection of one or more nodes running clustering software such as Docker Swarm.
+Nodes may or may not be homogeneous in their capabilities.
 
-###Base Image/Image
-A base OS image with *minimal set* of pre-installed packages that are needed for cluster
-management service to manage that node. For instance, for automated discovery of node a
-cluster membership service like Serf needs to be pre-installed in the base OS and started
-at node's bootup.
+###Image
+An operating system with a *minimal set* of pre-installed packages used by a node. For instance,
+node automated discovery requires a cluster membership service like [Serf](https://www.serfdom.io/)
+to be pre-installed and started on the node's operating system during bootstrapping.
 
 ###Bootstrap
-When used in context of node, it refers to the process of installing the node image and
-bringing up the node first time. It is sometime also referred to as node onboarding.
 
-When used in context of cluster, bootstrap refers to the process of adding first bootstrapped
-node to the cluster.
+**Node Bootstrap**: The process of installing the node image and booting the node for the first time.
+
+**Cluster Bootstrap**: The process of adding bootstrapped nodes to the cluster.
 
 ###Lifecycle
-A collection of well-defined states that a node or cluster goes through when certain events happen.
+A collection of well-defined states that a node or cluster transitions through based on system events.
+
+###Infra Service
+A userspace application or a kernel module that runs on a node and provides a support service to
+the clustering system software. Examples include plugins, drivers, or components for networking
+and storage, key/value stores, etc..
 
-###Infra App/Infra Service
-An userspace application or a kernel module that is used to provide one or more system services
-like plugins/drivers or controller components for networking and storage; key/value stores; etc.
+##Features
+
+Cluster Manager provides the following features. Not all features are implemented, or will be implemented in the initial release. Please review the [Roadmap](management/ROADMAP.md) or continue reading to understand individual feature status.
```

Comment:
@danehans 

Sorry I wasn't very clear. So the list here contains features like `Application definition, launch and monitoring` and `storage and network provisioning for user and infrastructure applications` which don't belong to `Cluster manager` per se. I had put them here earlier as I had thought that we could use this doc to showcase some of the offerings of `Contiv stack` in it's entirety.

But thinking more I think this causes confusion as the reader of the document is primarily looking for cluster manager feature here i.e. only the items under `Cluster management` bullet and the `Node discovery to streamline cluster expansion` bullet.

May be we can remove the `Application definition, launch and monitoring` and `storage and network provisioning for user and infrastructure applications` bullets, then the wording that you originally have will be accurate.